Source/Core/ClippingPlaneCollection.js

```diff
@@ -1,40 +1,66 @@
 define([
+        './AttributeCompression',
+        './Cartesian2',
         './Cartesian3',
         './Cartesian4',
+        './Math',
         './Check',
+        './ClippingPlane',
         './Color',
         './defaultValue',
         './defined',
         './defineProperties',
+        './destroyObject',
         './DeveloperError',
         './FeatureDetection',
         './Intersect',
         './Matrix4',
-        './Plane'
+        './PixelFormat',
+        './Plane',
+        '../Renderer/ContextLimits',
```

Comment:
This file is in `Core`, should it be including all these `Renderer` references?  Or should this file be moved to `Scene` or maybe `Renderer`?

@likangning93 for more on the Cesium stack, see https://cesium.com/blog/2015/05/26/graphics-tech-in-cesium-stack/